ClipVessel/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+#-----------------------------------------------------------------------------
+set(MODULE_NAME ClipVessel)
+
+#-----------------------------------------------------------------------------
+set(MODULE_PYTHON_SCRIPTS
+  ${MODULE_NAME}.py
+  )
+
+set(MODULE_PYTHON_RESOURCES
+  Resources/Icons/${MODULE_NAME}.png
```

Comment:
Please add a new icon. You can generate it with any AI image generator.

Reply:
Hi @lassoan, 

Sorry for the long time to address these issues. I have updated the module. Please let me know if there are any remaining issues.